packages/core/templates/resourse-list.ts.ejs

```diff
@@ -1,3 +1,3 @@
 export const ResourceNames = [<% for (const structureDef of fhir.resources) { -%>
```

Comment:
I beliebe this _almost_ already exists in the package as `DomainResourceTypes` - why would you need all Resources instead of just the DomainResources for this change?